packages/component-library/src/components/table-and-list/mt-data-table/mt-data-table.interactive.stories.ts

```diff
@@ -557,3 +557,93 @@ export const VisualTestEnableRowNumbering: MtDataTableStory = {
     await expect(canvas.getByText("Awesome Concrete Chair")).toBeInTheDocument();
   },
 };
+
+export const EmitOpenDetailsEventOnClickingEdit: SwDataTableStory = {
```

Comment:
This needs to be renamed to `MtDataTableStory` to fix the issue in the static analysis

Reply:
@jleifeld You're right. My bad, I am sorry